### ISSUE TITLE
SlevomatCodingStandard.Variables.UnusedVariable false positive with compact

### ISSUE THREAD

**Issue by arxeiss** [slevomat/coding-standard#1819]
Hi,

the sniff `SlevomatCodingStandard.Variables.UnusedVariable` is reporting all variables as unused, if they are used only in `compact` function.

Here variable `$students` is reported as unused. Strange is, it worked well for 8.20.0 and now after upgrade to 8.25.1 it reports those variables.

```php
public function index(): View
    {
        $this->authorize('parentList', Student::class);
        $students = $this->studentService->getListForParent(Auth::user())->get();

        return \view('students.list', \compact('students'));
    }
```
